tests/e2e/conftest.py

```diff
@@ -339,7 +353,6 @@ def run_once(
                 chunk = _get_chunk(child)
                 output += chunk
     finally:
-        output += _read_till_end(child)
```

Comment:
this blocked until the process finished. Added a test with `sleep infinity` as the last command.